src/login/tests/LoginPage.test.jsx

```diff
@@ -203,12 +204,12 @@ describe('LoginPage', () => {
   // ******** test alert messages ********
 
   it('should match login error message', () => {
-    const errorMessage = 'Email or password is incorrect.';
+    const errorMessage = 'An error has occurred. Try refreshing the page, or check your internet connection.';
     store = mockStore({
       ...initialState,
       login: {
         ...initialState.login,
-        loginError: { value: errorMessage },
+        loginError: { errorCode: INTERNAL_SERVER_ERROR },
```

Comment:
Should not we check here for `INCORRECT_EMAIL_PASSWORD = 'incorrect-email-or-password'` errorCode?